woocommerce/woocommerce-gateway-stripe
# feat: add express checkout settings

## Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1705
Adding the "Express checkout" section UI.

## Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- You should see the "Express checkout" section UI updated
- Toggling the checkboxes works
- Toggling the "Enable express checkouts" checkbox ensures that the other checkboxes are enabled/disabled

![2021-08-20 14 30 36](https://user-images.githubusercontent.com/273592/130284507-d8fbacf2-077e-4bc5-9ee2-99977c9262b9.gif)
